feedback.md

```diff
@@ -0,0 +1,32 @@
+Hi Kim!
+
+Many thanks for this awesome submission. I went through all (I hope) files and have a few follow-up questions - no coding needed, just conceptual stuff!
+
+Hope that's fine! :)
+
+# Worker:
+
+https://github.com/Burgestrand/kbc19/blob/master/lib/tasks/active_job.rake all good here! Clean and concise!
+
+# Jobs:
+
+https://github.com/Burgestrand/kbc19/blob/master/app/jobs/application_job.rb#L5: we may want to file this as a bug in rails/rails; if I remember correctly connection pool errors during deserialize would lead to discard!
+
+# Adapter:
+
+https://github.com/Burgestrand/kbc19/blob/master/app/jobs/google_pub_sub_adapter.rb
+
+- Maybe should go to lib? Unsure.
```

Comment:
Not a stranger to this — I just found it fitting that job-specific code be placed in `jobs/`. An upside is that `jobs/` is reloaded by default; although that doesn't matter since we use/configure the adapter in an initializer anyway.

Ideally all of this would be extracted into a gem, that way we could do all the cool constant autoloading (e.g. [Zeitwerk](https://github.com/fxn/zeitwerk), rails uses it too) and move implementation pieces (e.g. `Queue`) into separate files.